From 04b555b3cbc58a2632b205f1a9892dfa2e9d9f4a Mon Sep 17 00:00:00 2001
From: Cristian Iorga <cristian.iorga@intel.com>
Date: Tue, 15 Mar 2016 16:33:08 +0200
Subject: [PATCH 2/3] vendor_libs: create Android.mk

all-subdir-makefiles call works for only one
nested sublevels, so we need to add a default
Android.mk in order to be able to build libbt-vendor module.

BUG=28014054

Change-Id: I7cc004d2a4dd6216a80d13c6384d9dcde328323e
Signed-off-by: Cristian Iorga <cristian.iorga@intel.com>
---
 vendor_libs/Android.mk | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 vendor_libs/Android.mk

diff --git a/vendor_libs/Android.mk b/vendor_libs/Android.mk
new file mode 100644
index 0000000..5053e7d
--- /dev/null
+++ b/vendor_libs/Android.mk
@@ -0,0 +1 @@
+include $(call all-subdir-makefiles)
--
2.7.4
